### ISSUE TITLE
An error about InitFlux LoRATraining

### ISSUE THREAD

**Issue by linseasea** [kijai/ComfyUI-FluxTrainer#89]
![屏幕截图 2024-10-31 190332](https://github.com/user-attachments/assets/0eced4c8-cda0-4d38-b958-12c0cf38e4ed)

I was still using it normally last week and didn't update Comfyui.
There was an error this week, and I am very confident that I did not make any changes to the settings.
I even reset my Comfyui and plugins.

**Comment by kijai** [comment]
Check the last widgets of that node, the gradient checkpointing options changed at one point and it looks like the workflow is trying to use non-existing choice.

**Comment by linseasea** [comment]
Thank you very much for your efficient feedback.
I also noticed this small setting, it's really too hidden.
I've been searching for a long time.
This issue has been resolved.

I have some small questions to summarize:
1. I see that there is only num_depeaks here, but no setting for the number of loops.
2. If I have 2000 1024 * 1024 images, do you recommend using Lora for training or fine-tuning the large model?
3. How can I calculate the total number of steps based on the parameters of this script?
